source/about-compatibility.txt

```diff
@@ -58,3 +61,16 @@ labeled with major release versions of the driver.
 
 The check marks (✓) indicate that the code in the driver is fully
 compatible with the version of the language.
+
+Component Compatibility Tables
+------------------------------
+
+In the **Component Compatibility** tables, the columns indicate versions of
+the driver, and the rows indicate the component package names.
+
+The table cells indicate which versions of the component packages are
+compatible for the driver version specified in that column.
```

Comment:
S: AFAIK, things are generally compatible "with" (as stated in the next paragraph), not compatible "for"

```suggestion
compatible with the driver version specified in that column.
```

Reply:
Missed in my proofreading, thanks!